partcad-cli/src/partcad_cli/click/conftest.py

```
@@ -0,0 +1,21 @@
+
+
+@pytest.fixture
+def click_runner(capsys: pytest.CaptureFixture[str]) -> Iterator[CliRunner]:
+    """
+    Convenience fixture to return a click.CliRunner for cli testing
+    """
+
+    class MyCliRunner(CliRunner):
+        """Override CliRunner to disable capsys"""
+
+        def invoke(self, *args, **kwargs) -> Result:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)